website/static/css/style.css

```diff
@@ -372,13 +372,19 @@ a {
 }
 
 /* OSF-panel */
+
 .osf-panel {
     border: 1px solid #ddd;
-    height: 764px;
-    overflow: auto;
     display: block;
     position: relative;
-    overflow-x: hidden;
+	overflow-x: hidden;
```

Comment:
Spacing here is off. Also, if you revert it back to what it was before rather than moving/reordering the lines, you won't have the unnecessary diff and you'll be less likely to cause a merge conflict.